relays/bin-substrate/src/cli.rs

```diff
@@ -414,3 +444,31 @@ macro_rules! declare_chain_options {
 		}
 	};
 }
+
+#[cfg(test)]
+mod tests {
+	use std::str::FromStr;
+
+	use super::*;
+
+	#[test]
+	fn should_format_addresses_with_ss58_format() {
+		// given
+		let rialto1 = "5sauUXUfPjmwxSgmb3tZ5d6yx24eZX4wWJ2JtVUBaQqFbvEU";
+		let rialto2 = "5rERgaT1Z8nM3et2epA5i1VtEBfp5wkhwHtVE8HK7BRbjAH2";
+		let millau1 = "752paRyW1EGfq9YLTSSqcSJ5hqnBDidBmaftGhBo8fy6ypW9";
+		let millau2 = "74GNQjmkcfstRftSQPJgMREchqHM56EvAUXRc266cZ1NYVW5";
```

Comment:
It might be useful to have a testcase where the formatting fails

Reply:
Not sure I get what you mean, do you want to trigger an error coming from `FromStr` implementation? It will only fail if the address is simply not a valid address, but that's something that is tested in the substrate repo already.

Currently the implementation is able to read address with any SS58 prefix and allows the caller to reformat (`enforce_chain`) it to desired format. I've added a test for this.